workflows/prognostic_c48_run/runtime/diagnostics/tensorboard.py

```diff
@@ -0,0 +1,41 @@
+import io
+import logging
+from typing import Mapping
+
+import cftime
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import xarray as xr
+
+logger = logging.getLogger(__name__)
+
+
+def plot_to_image(figure):
+    """Converts the matplotlib plot specified by 'figure' to a PNG image and
+    returns it. The supplied figure is closed and inaccessible after this call."""
+    # Save the plot to a PNG in memory.
+    buf = io.BytesIO()
+    plt.savefig(buf, format="png")
+    # Closing the figure prevents it from being displayed directly inside
+    # the notebook.
+    plt.close(figure)
```

Comment:
Why does the close appear here and not in the context where the figure is created?  Maybe adjusting `savefig` above means it doesn't have to be closed here?

Reply:
I'll remove it.